cli-toolchain/graphql.md

```diff
@@ -1739,11 +1754,288 @@ mutation CreateProject {
 }
 ```
 
-> **Note** The **Project.team** resolver is configured to work with the defined connection.
+> **Note** The **Project.team** resolver is configured to work with the defined connection. This is done with a query on the Team table where `teamID` is passed in as the partition key.
```

Comment:
Do we need to say "partition key"? In the context of GraphQL you just need to know it's an argument.